packages/service-library/src/servicelib/fastapi/tracing.py

```diff
@@ -45,3 +79,19 @@ def setup_tracing(
     global_tracer_provider.add_span_processor(span_processor)
     # Instrument FastAPI
     FastAPIInstrumentor().instrument_app(app)
+
+    if HAS_AIOPG:
+        log.info("Attempting to add aiopg opentelemetry autoinstrumentation...")
```

Comment:
here the log makes sense, since it is before the call.
we have something that you could use though, which would automatically add a log of type info before and after whatever is in the context manager.
```python
with log_context(
            log,
            logging.INFO,
            msg=f"adding aiopg opentelemetry",
        ):
    AiopgInstrumentor().instrument()
```

Reply:
done thx